### DISCUSSION TITLE
Any way to access the remote camera using ReactJs

### DISCUSSION THREAD

**Discussion by AlanBinu007** [AlanBinu007/Realtime-Chat-Application-ReactJs#13] [Q&A] [answered]
Any way to access the remote camera using ReactJs

**Comment by DanMathew7** [comment, marked as answer]
I think we can access the cameras using IP address

**Reply by AlanBinu007** [reply]
Thanks.
I mean, the camera form outside form our network

**Reply by DanMathew7** [reply]
OK, then can this link helpfull : https://stackoverflow.com/questions/21679400/access-an-internal-network-webcam-from-outside-www

**Reply by AlanBinu007** [reply]
Let me check
